src/deb/control/control

```diff
@@ -0,0 +1,8 @@
+Package: runelite
+Version: [[version]]
+Section: games
+Priority: optional
+Architecture: all
+Maintainer: Adam <Adam@sigterm.info> 
+Description: RuneLite client for OldSchool RuneScape.
+Depends: openjdk-8-jre | openjdk-7-jre | openjdk-6-jre | oracle-java8-installer | oracle-java7-installer | oracle-java6-installer | jarwrapper
```

Comment:
RuneLite only works for Java 8+. So 7 and 6 should be removed, then add openjdk's and oracle's Java 9 and Java 10.
I'm not familiar with jarwrapper. What is it used for? Is it necessary? If so, shouldn't the depends line have a comma instead of a vertical pipe?